runtime/lua/vim/treesitter/dev.lua

```diff
@@ -530,8 +530,13 @@ function M.inspect_tree(opts)
   api.nvim_create_autocmd({ 'BufHidden', 'BufUnload', 'QuitPre' }, {
     group = group,
     buffer = buf,
-    once = true,
     callback = function()
+      -- don't close inpector window if source buffer
+      -- has more than one open window
+      if #vim.fn.win_findbuf(buf) > 1 then
+        return
+      end
+
       -- close all tree windows
       for _, window in pairs(vim.fn.win_findbuf(b)) do
         close_win(window)
```

Comment:
I think this now needs to `return true` to delete the autocmd.

Reply:
I think I see what you mean. Changing the return to true will make it throw ```E855: Autocommands caused command to abort``` on the last window again which I was trying to fix. Do you think not returning true will have side effects?

Reply:
You need to `return true` after all tree windows have been closed, not when the tree windows should be kept open.

Reply:
Oh nvm you meant a a return at the end right? That works